.golangci.yml

```diff
@@ -8,7 +8,7 @@ linters:
   enable:
     - bodyclose
     - deadcode
-    - depguard
+    # - depguard
```

Comment:
Any reason we wanted this commented out of the linter?

Reply:
It makes it easier for me to lint locally. For some reason, this line isn't enforced on CI, and I am not sure why we enforce it in the local lint config